src/components/Nav.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import CityFilter from './City/CityFilter';
+import './styles/Nav.css';
+
+const Nav = ({ onFilterChange }) => (
+  <header>
+    <nav className="flex">
```

Comment:
Great work so far 👍🏼 , but please as the previous reviewer requested do your best to replicate the design, add the titles of theviews and the icons missing on top of the page and you still can add the search input before the list of items 

![Capture d’écran du 2023-08-11 16-26-51](https://github.com/DevaWinner/airtrackr/assets/59707859/fa4e8459-95ff-426d-98f4-55deaa50e811)

Reply:
Thank you. Corrected as suggested.